src/Components/WebAssembly/Authentication.Msal/src/Interop/dist/.gitignore

```diff
@@ -1,2 +1,3 @@
-*.js
-*.js.map
+**/*.js
```

Comment:
Why are we committing the `dist` directory? Looks to be empty (aside from this) : https://github.com/dotnet/aspnetcore/tree/93a3b43a655af3fc2c0087ec0851f33e8c14d269/src/Components/WebAssembly/Authentication.Msal/src/Interop/dist

Reply:
It is because we have a .gitignore for not checking in build outputs.